app.js

```diff
@@ -23,6 +24,12 @@ app.use(
   }),
 );
 
+app.use((req, res, next) => {
+  res.setHeader('Access-Control-Allow-Origin', 'http://localhost:3000');
+  next();
+});
```

Comment:
Is this necessary? I believe the cors call above this one should be enough for the frontend to connect to backend but please test on your end without this app.use() call.